doc/releases/release-notes-2.3.rst

```diff
@@ -500,7 +500,23 @@ Bluetooth
 Build and Infrastructure
 ************************
 
-* <TBD>
+* Zephyr CMake package
+
+  * The zephyr repository now includes a Zephyr CMake package.
```

Comment:
capitalize 'zephyr', or, better, write: "the Zephyr main repository"

Reply:
fixed, using `The Zephyr main repository`